app/services/onlinetesting/phase2/EvaluatePhase2ResultService.scala

```diff
@@ -50,6 +52,13 @@ trait EvaluatePhase2ResultService extends EvaluateOnlineTestResultService[Phase2
       case _ => throw new IllegalStateException(s"Illegal number of phase2 active tests with results " +
         s"for this application: ${application.applicationId}")
     }
-    savePassMarkEvaluation(application, schemeResults, passmark)
+
+    CurrentSchemeStatusHelper.getSdipResults(application).map { sdip =>
+      if (application.isSdipFaststream) {
```

Comment:
Don't you already at this point, the application is sdip faststream?

I think if it is not sdipFastStream, function getSdipResults will return None.

Reply:
Here we are mapping the future - CurrentSchemeStatusHelper.getSdipResults method returns Future[List[SchemeEvaluationResult]] so if we are not sdip faststream we get back an empty list there is no option here

Reply:
It feels weird to return a List, when we know is either one or zero.

I can see it is very convenient to have it like a list, there may be a better way but probably not worthy to spend more time.